src/renderer/pages/project/script_editor.vue

```
@@ -437,4 +443,51 @@
+    images: updatedImages,
+  };
+
+  emit("update:mulmoValue", {
+    ...props.mulmoValue,
+    imageParams: updatedImageParams,
+  });
+};
+
+const updateImagePath = (imageKey: string, path: string) => {
+
+  const currentImages = props.mulmoValue?.imageParams?.images ?? {};
```

ESLint

[error] Delete `⏎`

(prettier/prettier)